backend/coins/eth/account.go

```diff
@@ -678,14 +681,30 @@ func (account *Account) FeeTargets() ([]accounts.FeeTarget, accounts.FeeTargetCo
 	return feeTargets, accounts.DefaultFeeTarget
 }
 
-// suggestedGasPrice returns the currently suggested gas price for the given fee target.
-func (account *Account) suggestedGasPrice(feeTargetCode accounts.FeeTargetCode) (*big.Int, error) {
+// gasPrice returns the currently suggested gas price for the given fee target, or a custom gas
+// price if the fee target is `FeeTargetCodeCustom`.
+func (account *Account) gasPrice(args *accounts.TxProposalArgs) (*big.Int, error) {
+	if args.FeeTargetCode == accounts.FeeTargetCodeCustom {
+		// Convert from Gwei to Wei.
+		amount, err := coin.NewAmountFromString(args.CustomFee, big.NewInt(1e9))
+		if err != nil {
+			return nil, err
+		}
+		gasPrice := amount.BigInt()
+		if gasPrice.Cmp(big.NewInt(0)) <= 0 {
```

Comment:
Maybe a nice future addition would be to compare the value to the economy fee target and, if it's smaller, warn users the fee might be too low.